tools/windows-node-installer/test/e2e/azure_test.go

```diff
@@ -151,11 +179,11 @@ func constructRequiredRules() (map[string]*requiredRule,
 	return requiredRules, nil
 }
 
-// setup does these prerequisite tests before running the tests.
+// loadInstanceInfo does these prerequisite tests before running the tests.
```

Comment:
Isn't this doing more than just loading instance information? Not sure if this is the correct name.